index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+<!-- The Links --> 
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="font-awesome/css/font-awesome.css"> 
+</head>
+<body>
+<!-- The Header -->   
+  <header class="header"> 
+  <div class="contect-info" >
+    <span class="time" >12:00</span>  
+    <span class="fa fa-battery-full"></span> 
+    <span class="fa fa-wifi"></span>
+    <span class="fa fa-signal"></span>
+  </div>
```

Comment:
- [ ] This content should be displayed by the user's phone and this should not be present on the portfolio page. This content is displayed in the Figma template only for purposes of making it look like a phone, but it should not be copied. Kindly remove this because it can be overlapped by the user's phone.